tests/e2e/default/transitions/create-user-for-contacts.create-user.wdio-spec.js

```diff
@@ -10,75 +8,25 @@ const loginPage = require('@page-objects/default/login/login.wdio.page');
 const contactsPage = require('@page-objects/default/contacts/contacts.wdio.page');
 const { BASE_URL } = require('@constants');
 const { cookieLogin } = require('@page-objects/default/login/login.wdio.page');
-const genericForm = require('@page-objects/default/enketo/generic-form.wdio.page');
+const createUserForContactsPage = require('@page-objects/default/enketo/create-user-for-contacts');
 
-const CONTACT_NAME = 'Bob_chw';
-const ADD_CHW_FORM = 'form[data-form-id="add_chw"]';
-
-const chwNameField = () => $(`${ADD_CHW_FORM} input[name="/add_chw/chw_profile/name"]`);
-const chwPhoneField = () => $(`${ADD_CHW_FORM} input[name="/add_chw/chw_profile/phone"]`);
-
-const setChwName = async (nameValue = 'Ron') => {
-  const name = await chwNameField();
-  await name.waitForDisplayed();
-  await name.setValue(nameValue);
-};
-
-const setChwPhone = async (phoneValue = '+40755696969') => {
-  const phone = await chwPhoneField();
-  await phone.waitForDisplayed();
-  await phone.setValue(phoneValue);
-};
-
-const submitAddChwForm = async ({
-  name: nameValue,
-  phone: phoneValue,
-} = {}) => {
-  await setChwName(nameValue);
-  await setChwPhone(phoneValue);
-  await genericForm.submitForm();
-};
-
-const district = utils.deepFreeze(
-  placeFactory
-    .place()
-    .build({ type: 'district_hospital' })
-);
-
-const settings = utils.deepFreeze({
-  transitions: { create_user_for_contacts: true },
-  token_login: { enabled: true },
-  app_url: BASE_URL
-});
-
-const settingsNoTransitions = utils.deepFreeze({
-  transitions: { create_user_for_contacts: false },
-  token_login: { enabled: true },
-  app_url: BASE_URL
-});
+describe('Create user when adding contact', () => {
+  const NEW_USERS = [];
+  const CONTACT_NAME = 'Bob_chw';
+  const FORM_NAME = 'add_chw';
 
-const offlineUser = utils.deepFreeze(userFactory.build({
-  username: 'offline_user_create',
-  place: district._id,
-}));
+  const district = utils.deepFreeze( placeFactory.place().build({ type: 'district_hospital' }) );
 
-const addChwAppForm = utils.deepFreeze({
-  _id: 'form:add_chw',
-  internalId: 'add_chw',
-  title: 'Add CHW',
-  type: 'form',
-  _attachments: {
-    xml: {
-      content_type: 'application/octet-stream',
-      data: Buffer
-        .from(fs.readFileSync(`${__dirname}/forms/add_chw.xml`, 'utf8'))
-        .toString('base64')
-    }
-  }
-});
+  const setSettings = (create_user_for_contacts) => utils.deepFreeze({
```

Comment:
Minor, but `set` seems incorrect to me for this function (since really this is just creating a new settings doc, but not really "setting" it anywhere).

IMO `getSettings` or `createSettings` would make more sense.

Reply:
Thank you for paying attention to the method's names, I will change it 🤩